### ISSUE TITLE
Remove useless console.log()

### ISSUE THREAD

**Issue by xbegault** [Foliotek/Croppie#134]
Hi,

can you remove console.log(), see : https://github.com/Foliotek/Croppie/blob/v2.1.0/croppie.js#L988

Thanks,

Xav

**Comment by JimmyMultani** [comment]
I believe that's there for debugging purposes. If you don't plan on needing it, consider minifying your JS or using the minified Croppie JS file.

**Comment by xbegault** [comment]
Yes ok thanks for the answer, but the console.log file exist in your min js : https://github.com/Foliotek/Croppie/blob/v2.1.0/croppie.min.js

By default and for example, gulp don't delete console.log().
